README.md

```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
     $ kitchen converge  # provisions/installs from chef cookbooks
     $ kitchen login     # ssh in to vm
 
+### MessyBrainz and ListenStore
+
+ListenStore is our interface to cassandra. It's bundled as part of this
+repository, but you must install it so that python can find it.
+
+    $ cd listenstore
+    $ python setup.py develop
+
+You can use `setup.py develop` to symlink the listenstore directory into
+site-packages (letting you make changes easily), or use `setup.py install`
+to install it. If you make any changes to listenstore you must install
+again.
```

Comment:
If you use "install" and make any changes to listenstore you must run install again.